client/html/layouts/basket/product/body-default.html

```diff
@@ -0,0 +1,114 @@
+<?php
```

Comment:
okay, I changed the header-default.html and the body-default.html to product-header-default.html and product-body-default.html. I also eddited the default.php so the $default matches the template files. Although, now I get an 'A non-recoverable error occured'. I'm not sure what I'm missing now?